README.md

```diff
@@ -183,7 +183,7 @@ test('my app works', async () => {
    - [`server.router`](#serverrouter)
    - [`server.start()`](#serverstart)
    - [`server.stop()`](#serverstop)
- - [Request](#request-1)
+ - [Request](#request)
```

Comment:
I believe this change is incorrect. `#request` points to the `server.inject()` argument named `request`, whereas `#request-1` points to the unrelated `Request` class.

I'm opening to changing the `server.inject()` argument name, by the way, if it would help.

Reply:
> I believe this change is incorrect. `#request` points to the `server.inject()` argument named `request`, whereas `#request-1` points to the unrelated `Request` class.

Ah I see, I guess the IDE didn't see this, perhaps because it's a GitHub-flavoured Markdown feature.

> I am open to changing the `server.inject()` argument name, by the way, if it would help.

I think it's a good idea.